### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by rannick**
Update the documentation all new changes

## PR checklist

- [x] This comment contains a description of changes (with reason)
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Make sure your code lints (`nf-core lint .`).
- [x] Documentation in `docs` is updated
- [x] `CHANGELOG.md` is updated
- [x] `README.md` is updated

**Learn more about contributing:** [CONTRIBUTING.md](https://github.com/nf-core/rnafusion/tree/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,157 @@
...
+Update to DSL2 and newer software/reference versions
+
+### Non-DSL2 related changes
+
+- replace gtf file with chr.gtf: Please confirm if this change is ok for the whole pipeline
+- remove subdirectory with params.genome as only GRCh38 is supported (not necessary but helped me see the output more clearly, can be reversed)
+- modification of output parameters to make sure every important file is published
+- additional build_references flag to trigger the workflow to build references
+- container for starfusion is changed and the absolute path to prep_genome_lib.pl is given
+  --fasta argument is not required when executing the workflow to build references
```
Comment (jemten) [inline CHANGELOG.md:17]: This line does not render properly

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,157 @@
...
+- increase cpu + memory for process STARFUSION_DOWNLOAD
+- ci test done on stubs of reference building subprocesses ensembl and arriba
+- **For the moment, the rnafusion workflow (to analyse data and not build the references) is removed as dependencies and links would have to be fixed in the rnafusion workflow for execution of the build references otherwise**
```
Comment (jemten) [inline CHANGELOG.md:20]: This sentence could use some clarification.

**Review by jemten — COMMENTED** [review]
Nice work Annick! A lot of text 😄

File: CHANGELOG.md (hunk 2 above)
Comment (rannick) [reply CHANGELOG.md]: Addressed in 7bad769

**Review by jemten — APPROVED** [review]
Hooray 🎉

**Pull request merged**